common/nodejs-utils/src/bin/update_node_inventory.rs

```diff
@@ -0,0 +1,168 @@
+// Required due to: https://github.com/rust-lang/rust/issues/95513
+#![allow(unused_crate_dependencies)]
+
+use anyhow::{Context, Result};
+use heroku_inventory_utils::{
+    checksum::Checksum,
+    inv::{read_inventory_file, Arch, Artifact, Inventory, Os},
+};
+use node_semver::Version;
+use serde::Deserialize;
+use sha2::Sha256;
+use std::{
+    collections::{HashMap, HashSet},
+    env, fs, process,
+};
+
+/// Updates the local node.js inventory.toml with versions published on nodejs.org.
+fn main() {
+    let inventory_path = env::args().nth(1).unwrap_or_else(|| {
+        eprintln!("Usage: update_inventory <path/to/inventory.toml>");
+        process::exit(2);
```

Comment:
Why do the exit codes in this binary jump all over the place? I see `2`, `1`, `4`, `6`, and `7`. Are these supposed to mean something?

Reply:
I think I did this in the original inventory binaries. I was thinking it'd be easier for the devs using it to trace errors back to their source. I don't think it was a particularly good idea, and I don't think it's used anywhere. Feel free to use the exit codes that makes sense.

Reply:
Updated to use exit code 1 consistently (same as other scripts) https://github.com/heroku/buildpacks-nodejs/pull/814/commits/278eaea37b1846d16b19dc16c5d792d7ad729112